youtube_dl/extractor/letv.py

```diff
@@ -239,3 +240,75 @@ class LetvPlaylistIE(LetvTvIE):
         },
         'playlist_mincount': 7
     }]
+
+
+class LetvCloudIE(InfoExtractor):
+    IE_DESC = 'LetvCloud'
+    _VALID_URL = r'http://yuntv\.letv\.com/bcloud.html\?(?P<id>.*)$'
+
+    _TESTS = [{
+        'url': 'http://yuntv.letv.com/bcloud.html?uu=p7jnfw5hw9&vu=467623dedf',
+        'md5': '26450599afd64c513bc77030ad15db44',
+        'info_dict': {
+            'id': 'p7jnfw5hw9467623dedf',
+            'ext': 'mp4',
+            'title': 'p7jnfw5hw9467623dedf',
+            'timestamp': 1445922586,
+            'upload_date': '20151027',
+        },
+        'params': {
+            'format': '[width=1920]',
```

Comment:
1920x1080 have the format id 52, which is the largest, test pass even without format params and _sort_formats.